crates/holochain_types/src/chain.rs

```diff
@@ -180,26 +194,41 @@ pub enum MustGetAgentActivityResponse {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum BoundedMustGetAgentActivityResponse {
     /// The activity was found.
-    Activity(Vec<RegisterAgentActivity>, ChainFilterRange),
+    Activity {
```

Comment:
I gotta ask, if warrants are not deterministic, how can they be included in this `must_get` call?

Reply:
Similar to the above, this is not the actual return value of any must_get call. The warrants are peeled off before the activity is returned.